### ISSUE TITLE
Document how to deal with cubes with beams

### ISSUE THREAD

**Issue by astrofrog** [radio-astro-tools/spectral-cube#413]
At the moment, when loading a cube in Jy/beam, the ``.unit`` attribute will give Jy and summing into a 1D spectrum also gives a spectrum in Jy which is confusing since this is actually (I think?) Jy/beam.

I think we should document how to make use of the radio_beam package, as well as potentially make ``.unit`` return Jy/beam. Furthermore, we should show how to convert a cube from Jy/beam to Jy/pixel, which at the moment I could not figure out how to do without calling e.g. proj_plane_pixel_area.

Finally, I think we should make radio_beam a required dependency so that all the beam warnings no longer appear by default.

**Comment by keflavich** [comment]
This is closely related to https://github.com/astropy/astropy/pull/5173.  Astropy doesn't support "beam" as a unit very well at the moment.  This is a real problem and a pretty big flaw in how spectral_cube writes files.

The main reason for not including radio_beam as a hard dependency is to encourage use of this package for non-radio data.

**Comment by astrofrog** [comment]
@keflavich - radio_beam is such a lightweight dependency that (once released) I don't think there would be any harm in including it in ``install_requires``. I doubt many users would even notice.

I thought ``radio_beam`` subclassed Unit, hence why I thought the Jy/beam could be preserved, but just saw it's a Quantity. So is the solution to push forward with astropy/astropy#5173?

**Comment by astrofrog** [comment]
@keflavich - just thinking out loud about this, is there a reason we can't just create a new unit every time we parse a FITS file and make it correspond to an area on the sky:

```
In [15]: beam1 = u.def_unit('beam', represents=0.1 * u.sr)

In [16]: beam1
Out[16]: Unit("beam")

In [17]: beam2 = u.def_unit('beam', represents=0.2 * u.sr)
```

Then it will look like 'beam' to the user but will behave as a solid angle, which is basically what it is.

**Comment by keflavich** [comment]
Beams already have that property; a `beam` object is steradian-equivalent.  The problem is that the astropy equivalency I wrote is a Jansky-Kelvin equivalency, while it should have been a Jy/sr-Kelvin equivalency.

**Comment by keflavich** [comment]
Until astropy#5173 is merged, #310 is the answer.  We'll probably need to do some astropy version checking because pre/post 5173 will be incompatible.

**Comment by keflavich** [comment]
Closed by #310.
